packages/dappmanager/src/modules/chains/drivers/ethereum2.ts

```diff
@@ -0,0 +1,145 @@
+import fetch from "node-fetch";
+import memoize from "memoizee";
+import { getPrivateNetworkAlias } from "../../../domains";
+import { urlJoin } from "../../../utils/url";
+import { InstalledPackageData } from "../../../types";
+import { ChainDataResult } from "../types";
+import { safeProgress } from "../utils";
+
+const beaconChainServiceName = "beacon-chain";
+const MIN_SLOT_DIFF_SYNC = 60;
+
+// Wait for Promises to resolve. Do not cache rejections
+// Cache for 1 hour, genesis and config should never change
+const fetchNodeSyncingMemo = memoize(fetchNodeSyncingStatus, {
+  promise: true,
+  maxAge: 3e6
```

Comment:
You MUST not cache the syncing result for 1 hour, it changes almost every second. This caching was here exclusively for genesis and config calls, which never change.